### ISSUE TITLE
Incorrect message property name for 'isnan' warning

### ISSUE THREAD

**Issue by ghost** [douglascrockford/JSLint#97]
In the check_relation function, there is a call to warn with a 'isnan' argument if the node contains NaN:

``` javascript
warn('isnan', node);
```

However, the message bundle does not contain a property named 'isnan'. The corresponding message appears in a property named 'isNaN':

``` javascript
bundle = {
    isNaN: "Use the isNaN function to compare with NaN."
};
```

Validating a line such as `var x = 5 > NaN;` in the 2012-01-13 edition of http://jslint.com/ results in this message:

```
Problem at line 1 character 13: isnan
```
